src/toil/provisioners/clusterScaler.py

```diff
@@ -626,18 +691,14 @@ def _getInfo(allMesosNodes, ip):
             nodeToInfo[node] = info
         return nodeToInfo
 
-    def shutDown(self, preemptable):
+    def shutDown(self):
         if self.stats:
             self.stats.shutDownStats()
         logger.debug('Forcing provisioner to reduce cluster size to zero.')
-        totalNodes = self.setNodeCount(numNodes=0, preemptable=preemptable, force=True)
-        if totalNodes > len(self.scaler.provisioner.getStaticNodes(preemptable)):  # ignore static nodes
-            raise RuntimeError('Provisioner could not terminate all autoscaled nodes. There are '
-                               '%s nodes left in the cluster, %s of which were statically provisioned' % (totalNodes, len(self.getStaticNodes(preemptable)))
-                               )
-        elif totalNodes < len(self.scaler.provisioner.getStaticNodes(preemptable)):  # ignore static nodes
-            raise RuntimeError('Provisioner incorrectly terminated statically provisioned nodes.')
-
+        for nodeShape in self.nodeShapes:
+            preemptable = nodeShape.preemptable
+            nodeType = self.nodeShapeToType[nodeShape]
+            totalNodes = self.setNodeCount(nodeType=nodeType, numNodes=0, preemptable=preemptable, force=True)
```

Comment:
pyflakes says `local variable 'totalNodes' is assigned to but never used`. It looks like the call to self.setNodeCount is necessary, but maybe we can just ignore the return value?